### PR TITLE
feat: ScheduledToastNotification

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #5890 (partly)

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
No ScheduledToastNotification implementation.

## What is the new behavior?
Fields that are "the must" are implemented for all platforms, and two more fields for Android.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
 For all platforms, I implemented fields that seems to be required (without any of it, ScheduledToastNotification has no sense).
For Android, I implemented two more fields, as I know that these fields can be implemented (because I implemented it over year ago in non-scheduled Toasts, see #2585).

 Implementing toast scheduler is more tricky. 

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		}
+
+#if __ANDROID__
+		public  NotificationMirroring NotificationMirroring { get; set; }
+
+		public global::System.DateTimeOffset? ExpirationTime { get; set; }
+#endif
+
+		public ScheduledToastNotification( XmlDocument content, global::System.DateTimeOffset deliveryTime) 
+		{
```
Comment (MartinZikmund) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:43]: Move the constructor to the top of the file to have the right ordering (constructors, events, properties, methods)

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+using Microsoft.Extensions.Logging;
+using Windows.Data.Xml.Dom;
+
+namespace Windows.UI.Notifications
+{
+
+	public partial class ScheduledToastNotification
+	{
+		public  XmlDocument Content { get; internal set; }
+		public global::System.DateTimeOffset DeliveryTime { get; internal set; }
```
Comment (MartinZikmund) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:12]: Remove `global::System.` and add `using System;` instead

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			}
+		}
+
+#if __ANDROID__
+		public  NotificationMirroring NotificationMirroring { get; set; }
+
+		public global::System.DateTimeOffset? ExpirationTime { get; set; }
+#endif
+
+		public ScheduledToastNotification( XmlDocument content, global::System.DateTimeOffset deliveryTime)
```
Comment (MartinZikmund) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:42]: Similar here, avoid `global::System.`. Also remove the space after `(`

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs (hunk 1 above)
Comment (pkar70) [reply src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:43]: Order was same as in "your" Generated :)

**Comment by pkar70** [comment]
Could it be accepted/merged? This is only structs/enums.

**Comment by pkar70** [comment]
Can build be restarted?

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
+using Uno.Logging;
```
Comment (jeromelaban) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:1]:
```suggestion
#nullable enable
using Uno.Logging;

```

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+			Content = content;
+			DeliveryTime = deliveryTime;
+		}
+
+		public XmlDocument Content { get; internal set; }
+		public DateTimeOffset DeliveryTime { get; internal set; }
+
+		private string _tag;
+
+		public string Tag
```
Comment (jeromelaban) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:23]: Can this be null in UWP ? If it does, let's mark it as nullable.

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+		{
+			get
+			{
+				return _tag;
+			}
+			set
+			{
+				_tag = value;
+				if (_tag.Length > 64)
```
Comment (jeromelaban) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:32]: Please add a null check.

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs (hunk 5 above)
Comment (pkar70) [reply src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:23]: Cannot be null - added test in setter.

File: src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+	public partial class ScheduledToastNotification
+	{
+
+		public ScheduledToastNotification(XmlDocument content, DateTimeOffset deliveryTime)
+		{
+			if(content is null)
+            {
+				// yes, UWP throws here ArgumentException, and not ArgumentNullException
+				throw new ArgumentException("ScheduledToastNotification constructor: XmlDocument content cannot be null");
+            }
```
Comment (jeromelaban) [inline src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs:21]: Please use the VS formatter to adjust the indentation and spacing.

Comment (pkar70) [reply src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs]: ^K D done.

**Comment by pkar70** [comment]
Can this simple PR get ajpinedam review? "MartinZikmund requested a review from ajpinedam 29 days ago"...

**Pull request merged**